### PR TITLE
Task-39386:[NEWF] US_AlertMessages_01: Alert messages for Tasks manag…

### PR DESCRIPTION
**Pull request by mkrout**
…ement

### CODE REVIEW

File: platform-ui-skin/src/main/webapp/skin/less/variables.less [hunk 1]
```diff
@@ -90,16 +90,16 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 // Accent colors
 // -------------------------
 @errorColor:        #bc4343;     // border error
-@errorColorLight:   #d07b7b;   // background error
```
Comment (SaraBoutej) [inline platform-ui-skin/src/main/webapp/skin/less/variables.less:93]: Why did you change these colors codes? they can be used by other applications.
if you want to use #ffffff color ya can use @baseBackground variable

**Review by SaraBoutej — CHANGES_REQUESTED** [review]

**Review by SaraBoutej — APPROVED** [review]

File: platform-ui-skin/src/main/webapp/skin/less/variables.less (hunk 1 above)
Comment (mkrout) [reply platform-ui-skin/src/main/webapp/skin/less/variables.less:93]: IMO it will be better to change these common alert colors so we will have alerts with the same colors on the whole platform
